testing/database.test.js

```diff
@@ -0,0 +1,64 @@
+const mongoose = require('mongoose');
+const { MONGO_URI } = require('../config');
+const Hostel = require('../dataseeding/Schema/Hostel');
+const User = require('../dataseeding/Schema/User');
+const Review = require('../dataseeding/Schema/Review');
+
+let db;
+beforeAll(async () => {
+  db = await mongoose.connect(MONGO_URI);
+});
+
+describe('Database collection', () => {
+  describe('Database connection', () => {
+    test('mLab database server connected', () => {
+      expect(db).toBeTruthy();
+    });
+  });
+
+  describe('Data checking', () => {
+    describe('Hostel collection', () => {
+      test('should exist at least one hostel record', async () => {
+        const hostelRecord = await Hostel.findOne({});
+        expect(hostelRecord._id).toBeTruthy();
+      });
+      test('each hostel record should have a name field', async () => {
```

Comment:
I think it's best to remain consistent with test names. For example, update this to read 'should have a name field for each hostel'.

Reply:
yeah i agree with that